json/classic/src/main/java/ch/qos/logback/contrib/json/classic/JsonLayout.java

```diff
@@ -289,4 +351,68 @@ public boolean isIncludeContextName() {
     public void setIncludeContextName(boolean includeContextName) {
         this.includeContextName = includeContextName;
     }
+
+    public boolean isIncludeClassAndMethod() {
+        return includeClassAndMethod;
+    }
+
+    public void setIncludeClassAndMethod(boolean includeClassAndMethod) {
+        this.includeClassAndMethod = includeClassAndMethod;
+    }
+
+    public boolean isIncludeFileLine() {
+        return includeFileLine;
+    }
+
+    public void setIncludeFileLine(boolean includeFileLine) {
+        this.includeFileLine = includeFileLine;
+    }
+
+    public boolean isNeedJsonFormatMessage() {
+        return needJsonFormatMessage;
+    }
+
+    public void setNeedJsonFormatMessage(boolean needJsonFormatMessage) {
+        this.needJsonFormatMessage = needJsonFormatMessage;
+    }
+
+    public boolean isIncludeExceptionOnlyMsg() {
+        return includeExceptionOnlyMsg;
+    }
+
+    public void setIncludeExceptionOnlyMsg(boolean includeExceptionOnlyMsg) {
+        this.includeExceptionOnlyMsg = includeExceptionOnlyMsg;
+    }
+
+    public boolean isNeedSubExceptionStack() {
+        return needSubExceptionStack;
+    }
+
+    public void setNeedSubExceptionStack(boolean needSubExceptionStack) {
+        this.needSubExceptionStack = needSubExceptionStack;
+    }
+
+    public int getMaxExceptionStackLength() {
+        return maxExceptionStackLength;
+    }
+
+    public void setMaxExceptionStackLength(int maxExceptionStackLength) {
+        this.maxExceptionStackLength = maxExceptionStackLength;
+    }
+
+    private static final String DEFAULT_SUFFIX = "......content truncated! real length is ";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/tony19/logback-contrib/file/2370960320/issues/source?bid=3222643&fileBranchId=3222647#l403)